src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java

```diff
@@ -265,14 +267,23 @@ private void setRowValues(RowMap row, ResultSet resultSet, Table table) throws S
 			// need to explicitly coerce TIME into TIMESTAMP in order to preserve nanoseconds
 			if (columnDefinition instanceof TimeColumnDef)
 				columnValue = getTimestamp(resultSet, columnIndex);
-			else if ( columnDefinition instanceof DateColumnDef)
+			else if (columnDefinition instanceof DateColumnDef)
 				columnValue = resultSet.getString(columnIndex);
+			else if (columnDefinition instanceof DateTimeColumnDef)
+				try {
+					columnValue = resultSet.getString(columnIndex);
```

Comment:
I can't remember on this, did we need to the try catch for the DateTime column or can that go away to be more like the DateColumnDef now that the code was added to the DateTimeColumnDef?